### PR TITLE
Fixed #441, keyborad now hides before displaying snackbar, progress bar for password change added

### LINKED ISSUE #441: [Comfirm Password] Few Bugs
### Steps to reproduce
- Go to `Settings` page

### Issue 1
- Add eye icons to all the password fields. 

### Issue 2
- Type some invalid password in new password (less than 8 characters), and `Done` button. 
- Nothing is visible. 
**Fix** : hide the keyboard, snackbar is not visible otherwise

### Issue 3
Show loader when API call is being made. Show a progress icon below the Done button

### Issue 4
Clear the password fields, after the password is changed successfully

### PR DESCRIPTION
**Pull request by anirudh8860**
# Description

Fixes #441 
Snackbar is displayed by hiding keyboard. 
Now progress is shown on "Done" button.

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by anirudh8860** [comment]
Hey @Swati4star can you check whether it meets requirements?

**Comment by anirudh8860** [comment]
Hey @Swati4star is it okay now?

**Comment by Swati4star** [comment]
@anirudh8860 What about eye icons to password field?

File: Android/app/src/main/java/io/github/project_travel_mate/SettingsFragment.java [hunk 1]
```diff
@@ -170,6 +177,7 @@ public boolean validatePassword(String passString) {
                 Snackbar snackbar = Snackbar
                         .make(mActivity.findViewById(android.R.id.content),
                                 R.string.invalid_password, Snackbar.LENGTH_LONG);
+                hideKeyboard();
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/SettingsFragment.java]: This function call is still here. Remove this one also.

File: Android/app/src/main/java/io/github/project_travel_mate/SettingsFragment.java [hunk 2]
```diff
@@ -15,12 +15,15 @@
...
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
 import android.widget.Button;
 import android.widget.EditText;
 import android.widget.LinearLayout;
 import android.widget.Switch;
 
 import com.airbnb.lottie.LottieAnimationView;
+import com.dd.processbutton.FlatButton;
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/SettingsFragment.java]: Also, do we need this import?

**Comment by anirudh8860** [comment]
Hey @Swati4star do tell if there is anything else.

File: Android/app/src/main/java/io/github/project_travel_mate/SettingsFragment.java (hunk 1 above)
Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/SettingsFragment.java]: @anirudh8860 This isn't addressed yet

**Comment by Swati4star** [comment]
@anirudh8860 hidekeyboard is still being called multiple times. You should also check once the changes

**Comment by anirudh8860** [comment]
Hey @Swati4star do tell if there is anything else.

**Comment by Swati4star** [comment]
add icon to old password field also

**Pull request merged**